tags/js/index.js

```diff
@@ -0,0 +1,71 @@
+// считает ссыслки
+function arrayCount(){
+    console.log(document.getElementsByTagName('a').length)
+}
+arrayCount()
+
+// считат картинки 
+function imgCount(){
+    console.log(document.getElementsByTagName('img').length)
+}
+imgCount()
+
+
+// вывод ширины первой картинки
+
+function getFirstImgWidth(){
+    console.log(document.getElementById('1').clientWidth)
+}
+getFirstImgWidth()
+
+
+// вывод ширины самой широкой картинки
+
+ 
+function getMaxImgWidth(){
+    const lengOne = document.getElementById('1').clientWidth
+    const lengTwo = document.getElementById('2').clientWidth
+    const lengThree = document.getElementById('3').clientWidth
+    let arImgWidth = [lengOne, lengTwo, lengThree];
+    let res = Math.max.apply(null, arImgWidth);
+    console.log(res)
```

Comment:
в методах всё хорошо, но нужно было результаты функций возвращать, чтобы можно было их записать на страницу:
```suggestion
    return res;
```